src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -485,9 +485,7 @@ private function buildTableRows($rows)
                 yield $this->fillCells($row);
 
                 if (isset($unmergedRows[$rowKey])) {
-                    foreach ($unmergedRows[$rowKey] as $row) {
-                        yield $row;
-                    }
+                    yield from $unmergedRows[$rowKey];
```

Comment:
That's going to be interesting, thank you for the review @stof!

Reply:
@ro0NL, wes the loop key was omitted here intentionally, or it can be used for generation delegating (asking as not too deep in the context here)?

Reply:
my change only preserved the current behavior back then... but LGTM.

the internals are a bit hard to follow here (we only use this for `foreach ($rows as $row)` though), but i believe it can cause duplicate keys due the yield above (which shouldnt hurt https://3v4l.org/Yaljq)

if tests pass im :+1:

Reply:
well, as TableRows does not care about the keys (and does not cast it to array, which might cause issues regarding duplicate keys), it is not an issue then.